src/main/java/me/contaria/seedqueue/mixin/client/MinecraftClientMixin.java

```diff
@@ -70,6 +70,13 @@ public abstract class MinecraftClientMixin {
     @Nullable
     public Screen currentScreen;
 
+    @Inject(method = "<init>", at = @At("TAIL"))
+    private void LogSystemInformation(CallbackInfo ci) {
+        if (Boolean.parseBoolean(System.getProperty("seedqueue.logSystemInfo", "true"))) {
+            SeedQueue.logSystemInformation();
+        }
+    }
+
```

Comment:
Please move this method below the core mixins, maybe before `runClientTasks`.
Also don't capitalize the "L" and format the annotation to use multiple lines, in this case it's not too bad but mixin annotations can get very long and as a result hard to read so I've commited to always using multiple lines in SeedQueue

Reply:
> Please move this method below the core mixins, maybe before `runClientTasks`. Also don't capitalize the "L" and format the annotation to use multiple lines, in this case it's not too bad but mixin annotations can get very long and as a result hard to read so I've commited to always using multiple lines in SeedQueue

done :D